README.md

```diff
@@ -221,6 +192,48 @@ XMR: 47gmN4GMQ17Veur5YEpru7eCQc5A65DaWUThZa9z9bP6jNMYXPKAyjDcAW4RzNYbRChEwnKu1H3
 ETN: etnkJXJFqiH9FCt6Gq2HWHPeY92YFsmvKX7qaysvnV11M796Xmovo2nSu6EUCMnniqRqAhKX9AQp31GbG3M2DiVM3qRDSQ5Vwq
 ```
 
+### Downloads <a name="downloads"></a>
+| Operating System | Download                                 |
+| ---------------- | ---------------------------------------- |
+| Windows 32       | https://releases.qwertycoin.org/get/cli/latest/win32 |
+| Windows 64       | https://releases.qwertycoin.org/get/cli/latest/win64 |
+| Mac 10.8 & Later | https://releases.qwertycoin.org/get/cli/latest/macos |
+| Linux 32         | https://releases.qwertycoin.org/get/cli/latest/linux32 |
+| Linux 64         | https://releases.qwertycoin.org/get/cli/latest/linux64 |
+| OpenBSD 64       | https://releases.qwertycoin.org/get/cli/latest/openbsd64 |
+| FreeBSD 64       | https://releases.qwertycoin.org/get/cli/latest/freebsd64 |
+| Linux ARM 64     | https://releases.qwertycoin.org/get/cli/latest/arm64 |
+| More Builds      | https://releases.qwertycoin.org |
```

Comment:
```suggestion
| Other platforms      | https://releases.qwertycoin.org |
```

Reply:
Just a suggestion. I think it may be easier to understand for non-technical people 🙂